packages/palette/src/Theme.tsx

```diff
@@ -82,11 +98,11 @@ export const themeProps = {
   // prettier-ignore
   /** Media queries to work with in web  */
   mediaQueries: {
-    xl: `(min-width: ${breakpoints.xl}px)`,
-    lg: `(min-width: ${breakpoints.lg}px) and (max-width: ${breakpoints.xl - 1}px)`,
-    md: `(min-width: ${breakpoints.md}px) and (max-width: ${breakpoints.lg - 1}px)`,
-    sm: `(min-width: ${breakpoints.sm}px) and (max-width: ${breakpoints.md - 1}px)`,
-    xs: `(max-width: ${breakpoints.sm -1}px)`,
+    xl: `(min-width: ${breakpoints.xl})`,
+    lg: `(min-width: ${breakpoints.lg}) and (max-width: ${parseInt(breakpoints.xl, 10) - 1})`,
```

Comment:
can we use `unitlessBreakpoints` here?

Reply:
I feel like we shouldn't just because it's a weird outlier solely for the benefit of the styled-grid component. If we get rid of that we wouldn't need it. 🤔